packages/experimental/Icon/src/Icon.tsx

```diff
@@ -6,6 +6,7 @@ import { SvgUri } from 'react-native-svg';
 import { mergeStyles } from '@fluentui-react-native/framework';
 import { stagedComponent, mergeProps, getMemoCache } from '@fluentui-react-native/framework';
 import { useTheme } from '@fluentui-react-native/theme-types';
+import { getCurrentAppearance } from '@fluentui-react-native/default-theme';
 
```

Comment:
This adds a direct dependency to default-theme for icon. I feel components shouldn't know about any specific theme?

Reply:
or this could be a function that is moved to the theme definition so every theme can use it? That maybe out of scope though

Reply:
Yeah I was being lazy. I'll move it to a theming folder under utils

Reply:
Ah, I didn't realize it would require creating a new package 😅, that does feel heavy for such a simple change.
IIRC, the purpose of the default theme having getCurrentAppearance() was because the default theme could override the appearance instead of what the system defined. 

I feel moving that functionality is still beneficial and better than having a component dependent on a specific theme. Could it not go in the `@fluentui-react-native/theme` package under `packages/experimental/theme`? But this is beginning to feel out of scope so I think I'll be fine with whatever you feel works best!

Reply:
Thanks for the suggestion! Taking a look, the `experimental/theme` feels "downstream" of `default-theme` (if that makes sense), and `default-theme` still uses `getCurrentAppearance()`, so I think keeping it close to that makes more sense compared to moving it under `experimental/theme`. I'm okay with having a utils folder under theming, might come in handy in the future.